uwmac0081.xml

```diff
@@ -1265,6 +1280,21 @@
                 </c02>
                 <c02>
                     <did>
+                        <unittitle>Fine Arts Alumni News Articles, <unitdate>1996,</unitdate>
+                            <unitdate>undated</unitdate></unittitle>
+                        <container type="box" label="Box">28</container>
+                        <container type="folder" label="Folder">2</container>
+                    </did>
+                </c02>
+                <c02>
+                    <did>
+                        <unittitle>Fine Arts Budget, <unitdate>1997-1998</unitdate></unittitle>
```

Comment:
I think I would file this under B for budget, unless "Fine Arts" is a key part of the description and not a recapitulation of the unit name